### PR TITLE
Msi unified docs

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by miczyg1** [comment]
Basically, it is sufficient to review only this commit: https://github.com/Dasharo/docs/pull/624/commits/fb06de89faa29fed5e386faf4d0b14853dfbf348

**Comment by miczyg1** [comment]
The comments do not appear if I paste just the commit hash, maybe when I link to commit from PR it will be different: https://github.com/Dasharo/docs/pull/624/commits/e31d499c3b2bd6fe02b524c0f0dafdaeda3d5d36

**Review by mkopec — CHANGES_REQUESTED** [review]

File: docs/unified/msi/hcl.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    found in [MSI PRO Z690-A Hardware Matrix](../../variants/msi_z690/hardware-matrix.md)
+    and [MSI PRO Z790-P Hardware Matrix](../../variants/msi_z790/hardware-matrix.md)
+    documentation.
+
+    Legend:
+
+    * GPU name: the full name of GPU including vendor and model name.
+    * Memory size: total amount of GPU memory declared by vendor.
+    * Memory type: GPU's type of memory.
+    * Bandwidth: GPU's bandwidth.
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/unified/msi/hcl.md:193]: @mkopec sometimes bandwidth is expressed in PCIe link width and sometimes in GB/s, dunno if it is memory or PCIe bandwidth of all the links...

File: docs/unified/msi/hcl.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+    | GPU name         | Memory size | Memory type  | Bandwidth | PCI-E Gen | Multi-Graphics Technology | Results                |
+    |:----------------:|:-----------:|:------------:|:---------:|:---------:|:-------------------------:|:----------------------:|
+    | Nvidia GeForce GTX 1060   | 3072 MB  | GDDR5  | 192GB/s   | Gen3      | 1                         | [Qubes HCL reports][1] |
+    | MSI Radeon RX 6950 XT     | 16 GB    | GDDR6  | 576GB/s   | Gen4      | 1                         | |
+    | EVGA NVidia RTX 2080      | 8 GB     | GDDR6  | 448GB/s   | Gen3      | 1                         | |
+    | PNY NVidia RTX A5000      | 24 GB    | GDDR6  | 768GB/s   | Gen4      | 1                         | |
+    | Nvidia GeForce GTX 1080TI | 11264 MB | GDDR5X | x16       | Gen3      | 1                         | [Qubes HCL reports][2] |
+    | MSI Radeon RX 6500 XT MECH 2X 4G OC | 4 GB     | GDDR6  | x4 (x16 connector) | Gen4      | 1                         | Works only on Dasharo v1.1.0 or newer |
+    | MSI GeForce RTX 3060 GAMING Z TRIO LHR | 12 GB     | GDDR6  | x16   | Gen4      | 1                         | |
```
Comment (miczyg1) [inline docs/unified/msi/hcl.md:208]:
```suggestion
    | MSI Radeon RX 6500 XT MECH 2X 4G OC | 4 GB     | GDDR6  | 180GB/s   | Gen4      | 1                         | Works only on Dasharo v1.1.0 or newer |
    | MSI GeForce RTX 3060 GAMING Z TRIO LHR | 12 GB     | GDDR6  | 358GB/s   | Gen4      | 1                         | |
```

File: docs/unified/msi/hcl.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/unified/msi/hcl.md:193]: We will also need this fix: https://github.com/Dasharo/docs/pull/624/files#r1280687832

[the remark this answers is not in the document]
Comment (mkopec) [reply docs/unified/msi/hcl.md]: Ah, just noticed the x16 and x4 below. Those are PCIe links indeed. But the values expressed in GB/s match these cards' memory bandwidth specs.

Doesn't really make sense to keep these in the same column.

[the remark this answers is not in the document]
Comment (mkopec) [reply docs/unified/msi/hcl.md]: Thanks

**Review by mkopec — APPROVED** [review]

**Review by macpijan — APPROVED** [review]

**Pull request merged**